ec2snap/ec2snap

```diff
@@ -647,6 +656,12 @@ def main():
             print("Don't have permission to read credentials file")
             sys.exit(1)
 
+    if arg.add_tags is not None and len(arg.add_tags) > 0:
+        for elem in arg.add_tags[0]:
+            if elem.find(':') == -1:
+                print("Tag '%s' has wrong format, missing separator ':'" % elem)
```

Comment:
As setup_log() return an instance of the logger, should be great to use it to log error messages instead of a print.
What do you think ?